manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md

```diff
@@ -60,32 +99,77 @@ PUT _ilm/policy/my_policy
 1. Roll over the index when it reaches 25GB in size
 2. Delete the index 30 days after rollover
 
-
+::::{important}
+The rollover action implicitly always rolls over a data stream or alias if one or more shards contain 200000000 or more documents. Normally a shard will reach 25GB long before it reaches 200M documents, but this isn’t the case for space efficient data sets. Search performance will very likely suffer if a shard contains more than 200M documents. This is the reason for the built-in limit.
+:::
 ::::
 
+## Create an index template to apply the lifecycle policy [apply-policy-template]
 
-::::{important}
-The rollover action implicitly always rolls over a data stream or alias if one or more shards contain 200000000 or more documents. Normally a shard will reach 25GB long before it reaches 200M documents, but this isn’t the case for space efficient data sets. Search performance will very likely suffer if a shard contains more than 200M documents. This is the reason of the builtin limit.
-::::
+To use a lifecycle policy that triggers a rollover action, you need to configure the policy in the index template used to create each new index. You specify the name of the policy and the alias used to reference the rolling indices.
```

Comment:
Needs Fleet-etAll disclaimers like "don't create your own edit the `@custom` part [described over here](https://www.elastic.co/guide/en/fleet/8.18/data-streams.html#data-streams-index-templates-edit)"  which ["create as needed" example](https://www.elastic.co/docs/solutions/observability/logs/logs-index-template-reference#custom-logs-template-edit)

Reply:
I added a note at the top of the doc, so that readers will know this page is specifically about ILM and not data stream lifecycles.

<img width="492" alt="screen27" src="https://github.com/user-attachments/assets/dbab5032-ebd3-4d1b-a1d3-c7292f4679d8" />